data/campaigns/Sceptre_of_Fire/scenarios/8_The_Dragon.cfg

```diff
@@ -264,7 +264,7 @@
         {RANDOM_MERCENARY "Dwarvish Lord" 2 13 ()}
         {RANDOM_MERCENARY "Dwarvish Lord" 2 14 ()}
         {RANDOM_MERCENARY "Dwarvish Dragonguard" 2 15 ()}
-        {RANDOM_MERCENARY "Dwarvish Dragonguard" 2 16 ()}
+        {RANDOM_MERCENARY "Dwarvish Dragonguard" 1 12 ()}
```

Comment:
I assume you playtested this and all of the other bugfixes/tweaks?

Reply:
Yes, I did.

The only change I have no idea how to test (without a lot of effort) is for [ab27fd8](https://github.com/wesnoth/wesnoth/pull/4882/commits/ab27fd85e62601fc5d2477b0ca4d10a51586700c) (scenario 4) - I am certain the code works for castles, but I'm not sure if it works for keeps.